src/web/templates/admin/accounts/tab.html

```diff
@@ -71,7 +67,53 @@
 {% block cards_container_cards %}
     {% if client.can_manage_accounts %}
         {% if admin_event.predefined_accounts %}
-            #TODO documentation
+            <div class="mx-auto" style="max-width: 500px;">
+                <h5 class="mt-2 my-5 text-center">
+                    {{ _('Staff default configuration') }}
+                </h5>
+                <p>
+                    {{ _(
+                        'Sharly Chess reaches its full potential when used as a '
+                        'server, with other devices connected to the same network.'
+                    ) }}
+                </p>
+                <p>
+                    {{ _(
+                        'By default, other devices can only interact with public screens. '
+                        'This tab allows you to configure this behavior.'
+                    ) }}
+                </p>
+                <p>
+                    {{ _(
+                        'You can create password-protected accounts to delegate tasks '
+                        'to other members of the staff (ex: other arbiters, organizer).'
+                    ) }}
+                </p>
+                <p>
+                    {{ _('Before configuring accounts, you have to be aware of a few security warnings:') }}
+                    <ul>
+                        <li class="mb-1">{{ _(
+                            'Some accounts can have very high access levels, making '
+                            'them dangerous in the hands of an unaware or malicious users.'
+                        ) }}</li>
+                        <li class="mb-1">{{ _(
+                            'Sharly Chess using the local network, all the traffic is public. '
+                            'This means anyone connected to the same network could easily gain '
+                            'access to an account without needing to type in the password.'
+                        ) }}</li>
+                        <li class="mb-1">{{ _(
+                            'If you can\'t control who has access to the network you '
+                            'are connected to, it is highly recommended to not create any account.'
+                        ) }}</li>
+                        <li>{{ _(
+                            'To learn how to control your network, consult '
+                            '<a href="%(link)s" target="_blank">our documentation</a>.',
+                            link=macros.doc_url('security')
+                        ) }}</li>
+                    </ul>
+                </p>
+                <p>{{ _('Click the [Get Started] button to configure accounts.') }}</p>
```

Comment:
To be consistent with other similar screens.
```suggestion
                <p>{{ _('Create an account now to get started.') }}</p>
```

Reply:
"Get started" does not create an account, so it feels weird to have that. I think the button is self explanatory, removing this sentence feels ok if it disturbs (the conclusion would be to send users to our doc, seems nice)